core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/Benchmark.scala

```diff
@@ -99,18 +99,31 @@ class Benchmark(
 
     val firstBest = results.head.bestMs
     // The results are going to be processor specific so it is useful to include that.
-    out.println(RuntimeUtil.getJVMOSInfo.mkString("\n"))
+    val jvmInfo = RuntimeUtil.getJVMOSInfo
+    out.printf(s"%-26s ->   %-30s \n","JVM Name", jvmInfo("jvm.name"))
```

Comment:
let's make it more standard  `:` instead of `->`

Reply:
Corrected